api/v1alpha1/jobset_types.go

```diff
@@ -74,6 +74,20 @@ type JobSetStatus struct {
 
 	// Restarts tracks the number of times the JobSet has restarted (i.e. recreated in case of RecreateAll policy).
 	Restarts int `json:"restarts,omitempty"`
+
+	// ReplicatedJobsStatus track the number of PodsReady for each replicatedJob.
```

Comment:
```suggestion
	// ReplicatedJobsStatus track the number of JobsReady for each replicatedJob.
```

You are getting the number of jobs that are ready.